README.md

```diff
@@ -1,27 +1,22 @@
-# The Rust Programming Language
+<div align="center">
+  <img alt="The Rust Programming Language: A language empowering everyone to build reliable and efficient software" src="https://www.rust-lang.org/static/images/rust-social-wide.jpg" width="512" height="256">
```

Comment:
This doesn't need to all be on one line like it currently is, how about
```suggestion
  <img alt="The Rust Programming Language: A language empowering everyone to build reliable and efficient software"
    src="https://www.rust-lang.org/static/images/rust-social-wide.jpg"
    width="512" height="256">
```

Maybe the other way around? :thinking: